Header header

uint32 r
uint32 g
uint32 b

string status
